test/spec/data/facet_spec.js

```diff
@@ -258,5 +258,24 @@ define(['test/mock', 'lodash'], function(mock, _) {
         });
       });
     });
+    describe('on uiClearFacets', function() {
+      beforeEach(function() {
+        this.component.trigger('config', mock.config);
+        this.component.trigger('data', mock.data);
+      });
+      it('emits a "dataFacets" event with no facets selected', function() {
+        this.component.trigger('uiClearFacets', {"facet" : "commudddnity"});
```

Comment:
I agree. What i'm trying to do is set the underlying 'selected' map that determines what is selected/unselected. It looks like the these classes went through some renovation lately - I used to be able to query like a this.component.attributes or something like that to get a selected hash.

How would you recommend this be triggered?